src/main/resources/api.yaml

```diff
@@ -58,7 +58,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/NoteSummaryDto'
-  /notes/recent:
+  /notes/recent: #todo we should remeber to add userId when it will exists
```

Comment:
I'm not sure about this. We could do it this way, but are we planning to allow users to view others recently viewed notes? For other endpoints we'll need to check if the user has permission to access the resource, so we could also extract userId from the token here. WDYT?

Reply:
right, we should allow this